### PR TITLE
Closing etc/os-release

### PR DESCRIPTION
**Pull request by gsegatti**
The os-release files raises a warning message when conducting unit tests with the __init__.py file, even though the file is in read mode. Below, the error message:

> home/gabrielsegatti/mkosi/mkosi/__init__.py:167: ResourceWarning: unclosed file <_io.TextIOWrapper name='/etc/os-release' mode='r' encoding='UTF-8'>
>    return dict(f(*args, **kwargs))
> ResourceWarning: Enable tracemalloc to get the object allocation traceback

In order to suppress this message, I've added a context manager to the code.

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -5631,7 +5632,6 @@ def load_distribution(args: argparse.Namespace) -> argparse.Namespace:
 
 def parse_args(argv: Optional[List[str]] = None) -> Dict[str, argparse.Namespace]:
     """Load default values from files and parse command line arguments
-
```
Comment (DaanDeMeyer) [inline mkosi/__init__.py:5634]: An unrelated whitespace change slipped in here. Can you remove that from the commit?

**Review by DaanDeMeyer — APPROVED** [review]

**Comment by behrmann** [comment]
LGTM. No need to burn more CI on this. Merging.
